assets/js/main.js

```diff
@@ -312,24 +312,24 @@ function search() {
             var output = '';
             data.results.forEach(function (post, index) {
                 var tooltipDescription = '';
-                var highlightedTitle = '';
+                var searchValueRegex = new RegExp(`(${searchValue})`, 'ig');
+                var highlightedTitle = post.title.raw.replaceAll(
+                    searchValueRegex,
+                    `<mark>$1</mark>`
+                );
```

Comment:
For this I think you can switch on `snippet` for title too.

Do watch out for performance and disable unneeded fields - we can discuss that further in some call.